app.py

```diff
@@ -15,6 +15,8 @@
 from database import db, BTUser, Profile
 from forms import LoginForm, RegisterForm
 
+bp = flask.Blueprint("bp", __name__, template_folder="./build")
```

Comment:
There is already a blueprint created in `setup.py`. This should be removed and imported from setup if absolutely necessary.